server/controllers/entries.js

```diff
@@ -1,4 +1,5 @@
 import data from '../models/data';
+import { runInDebugContext } from 'vm';
```

Comment:
Why is this here?

Reply:
i think i clicked it by mistake. will rectify now